README.md

```diff
@@ -26,7 +26,7 @@ golint prints out style mistakes.
 
 Golint differs from govet. Govet is concerned with correctness, whereas
 golint is concerned with coding style. Golint is in use at Google, and it
-seeks to match the accepted style of the open source Go project.
+seeks to match the accepted style of the open source Go project, [as described here](https://github.com/golang/go/wiki/CodeReviewComments).
```

Comment:
How about this sentence at the end of the paragraph?

```
Golinter's style guidelines often come from [Effective Go](https://golang.org/doc/effective_go.html) or [golang.org/wiki/CodeReviewComments](golang.org/wiki/CodeReviewComments).
```

 Looking at the errors generated from the linter, there are the two style guidelines that are roughly used.